source/documentation/tools/package-management.md

```diff
@@ -6,32 +6,100 @@ There are multiple package managers available for RStudio, JupyterLab & Airflow
+
+The basic renv commands are:
+
+| Command | Description |
+|------------------|---------------------------------------------|
+| `renv::init()`      | first time a project is created |
+| `renv::install()`   | install new packages |
+| `renv::snapshot()`  | save a description of packages to renv.lock |
+| `renv::restore()`   | install packages to match renv.lock |
+
 
-For a full guide to installing packages, workflow and installing custom packages please see the [introduction to renv](https://rstudio.github.io/renv/articles/renv.html).
+The following gives an overview of these basic renv commands.
+For more details check out the Coffee and Coding [video](https://web.microsoftstream.com/video/3ec54ac3-473c-4268-9d54-9f7096338824) and [slides](https://github.com/moj-analytical-services/Coffee-and-Coding/tree/master/2022-05-04%20Introduction%20to%20renv%20package%20management), or for a full guide to installing packages, workflow and installing custom packages please see the [introduction to renv website](https://rstudio.github.io/renv/articles/renv.html).
 
+### Getting started
 Before you start installing packages using renv, you need to enable it in RStudio. You do this by navigating to the Tools menu and going through the following steps:
 
 Tools -> Project options -> Environments and click on the tick box “Use renv with this project” then press OK.
 
-Basic commands to follow to install packages for `renv` are:
+The first time you use renv, you may be asked to consent to some changes it makes to the way packages are installed - please select yes to this.
+
+### Starting a new project with renv or adding renv to an existing project
+
+Basic commands to follow to install packages for renv are:
 
 ```r
 
-## If you are starting a fresh repository, run this:
+# install renv (if not already installed)
+install.packages(“renv”)
+
+# If you are starting a fresh repository, run this:
 renv::init(bare = TRUE) 
  
-## or if you are starting a fresh repository but would like to move your existing packages over to renv:
+# or if you are starting a fresh repository but would like to move your existing packages over to renv:
 renv::init()
 
-# then to install a package:
-renv::install("packagename")
 ```
 
+If you previously used a different package management system (like Conda or packrat) remove any configuration files for these systems from your R files first.
+
+Then ensure you have committed and pushed the relevant files (.Rprofile, renv.lock, and renv/activate.R) to your github repository.
+
+Now you are ready to work on your project!
+
+### Working on a renv project
+
+You can work on your project as normal now, but when you install new packages and want to save the state of your package environment you must "snapshot" your packages.
+
+For instance, if you wanted to install `dplyr` and then update your package environment then the process would be:
+
+```r
+
+renv::install("dplyr") 
+ 
+# snapshot your project
+renv::snapshot()
+
```

Comment:
Include a tidbit about installing specific package versions?
`renv::install("dplyr@0.8.5")` for example

Reply:
I've included this